example/src/PickerExample.tsx

```diff
@@ -17,6 +17,7 @@ function BasicPickerExample() {
     </Picker>
   );
 }
+export BasicPickerExample;
```

Comment:
This line seems to be causing syntax error. 
Could you add `export` at the beginning of line 7 instead?